### ISSUE TITLE
Show difference between time signature and the notes in the measure

### ISSUE THREAD

**Issue by BigHonkinDaddy** [powertab/powertabeditor#323]
It'd be great if it would calculate the total note duration for a given measure and display it to me, perhaps in bold or red or something if I'm over or under the time signature.

**Comment by cameronwhite** [comment]
This is somewhat related to #219

**Comment by BigHonkinDaddy** [comment]
Just an idea...  this would address issue #60, too.

![PTE enhancement 323](https://user-images.githubusercontent.com/77425146/105106139-8431bd80-5a83-11eb-949d-03ee9659bb9e.png)

**Comment by cameronwhite** [comment]
Yeah, I think the status bar is the right place for it. IIRC the original v1.7 had the total beat count vs time signature in the status bar too.

**Comment by BigHonkinDaddy** [comment]
Wow, I hadn't noticed that in 1.7 before... I just checked it and you're right... it's similar to what I showed above.  Rather than showing the note duration for the current selection, it shows the total note duration in the current measure and whether the notes are under ("-") or over ("+") the Time Signature.

![PTE enhancement 323 pic2](https://user-images.githubusercontent.com/77425146/105111999-15a72c80-5a90-11eb-9677-2d1515b6990f.png)

**Comment by Ebdim9th** [comment]
Wow. I never noticed that before either. Btw, one of the best things about Powertab is its flexibility. It doesnt force me to be correct when I'm trying to iron out some uncertainties about the way I want to time a measure or two, and on playback it doesnt skip past the notes that are ocer the time signature but plays them, so I can at least know if I'm getting into the ballpark or not...

**Comment by BigHonkinDaddy** [comment]
I think the best of both worlds would be...  if there's a highlighted selection of notes, show the number of beats by those selected notes.  Otherwise, show the over/under for the current measure.  Any help, though, on counting notes and time would be a huge benefit for me, and probably others too.

**Comment by Ebdim9th** [comment]
maybe a toggle  for completing measures you've already outlined w bars, based on the time signature, with additional placeholder replaceable rests, but without causing the MIDI playback cursor to skip over anything like other programs do, because i might find out i over or undershot because 3/4 actually winds up sounding better than 4/4 or 5/4 rather than 4/4 in that measure/measures.
